src/solana/utils/getLatestBlockHash.ts

```
@@ -0,0 +1,58 @@
+      } else {
+        // Default to using the existing central client
+        client = rpc;
+      }
+    }
+
+    switch (method) {
+      case "getSlot":
+        return await client.getSlot().send();
+
+      case "getLatestBlockhash":
+        const { value: latestBlockhash } = await client.getLatestBlockhash().send();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)